File: Assignment No.38/Program3.java
/* Problem statement : Write a program which accept one number from user and Toggle 7 bit of that number if it is on.return modified number.
Input : 137
Output : 201
*/
import java.util.*;

class Program3
{
	public static int ToggleBit(int iNo)
	{
		int iResult=0;
		int iMask=0x00000040;

		return (iNo ^ iMask);
		
		
	}
	public static void main(String args[])
	{
		Scanner sobj=new Scanner(System.in);
		int iNo=0;
		System.out.println("Enter the Number :");
		iNo=sobj.nextInt();

		int iRet=0;

		iRet=ToggleBit(iNo);
		System.out.println("Modified Number :"+iRet);
	}
}

File: Assignment No 3/Program3.c
N//Write a program which accept number from user and print even factor of that number on screen.
//Input : 36
//Output :  2  6  12  18
#include<stdio.h>

void DisplayFactor(int iNo)
{
	if(iNo <= 0)
	{
		iNo=-iNo;
	}
	int i=0;

	for(i=1;i<=iNo;i++)
	{
		if(((iNo%i)==0)&&((i%2)==0))
		{
			printf("%d\t",i);
		}
		
	}
}
int main()
{
	int iValue=0;
	printf("Enter number\n");

	scanf("%d",&iValue);
	DisplayFactor(iValue);
	return 0;
}